fpdf/html.py

```diff
@@ -738,11 +747,10 @@ def error(self, message):
 class HTMLMixin:
     HTML2FPDF_CLASS = HTML2FPDF
 
-    def write_html(self, text, *args, **kwargs):
-        """Parse HTML and convert it to PDF"""
-        kwargs2 = vars(self)
-        # Method arguments must override class & instance attributes:
-        kwargs2.update(kwargs)
-        h2p = self.HTML2FPDF_CLASS(self, *args, **kwargs2)
-        text = html.unescape(text)  # To deal with HTML entities
-        h2p.feed(text)
+    def __init__(self):
+        warnings.warn(
```

Comment:
Could you please add an extra unit test ensuring that this warning is raised as expected?

Reply:
You also need to make sure that any initialization arguments are passed on correctly:
```python
    def __init__(self, *args, **kwargs):
      super().__init__(*args, **kwargs)
      raise ...
```

Reply:
it's fixed at f0ef001116c48ac221fd2af92c8aaf1897a35011

Reply:
> Should I have added the `raise` keyword?

No, sorry, I was thinking about "raising" the warning there... 🙄